packages/react-components/src/components/DetailsCard/DetailsCard.module.scss

```diff
@@ -16,38 +18,39 @@ $base-class: 'details-card';
     border-bottom: 1px solid var(--border-basic-tertiary);
   }
 
-  &__button {
+  &__label-wrapper {
     display: flex;
     position: relative;
     align-items: center;
     justify-content: space-between;
-    transition: all 0.2s ease;
+    transition:
+      opacity var(--transition-duration-fast-1) ease-out
+        var(--transition-duration-moderate-1),
+      padding 0s ease-out var(--transition-duration-moderate-1),
+      max-height 0s ease-out;
```

Comment:
I think we can try to extract this into separate SCSS mixin function and use here and in other places below